build/loaders/convert-md-to-doc.js

```diff
@@ -150,8 +152,14 @@ async function convertMd2ComponentDocumentation (
     components = tokens[componentsIndex].text
     components = components
       .split('\n')
-      .map((component) => component.trim())
-      .filter((component) => component.length)
+      .map((component) => {
+        const textArr = component.split(':')
+        return {
+          name: textArr[0].trim() ? textArr[0].trim().split(',') : null,
+          path: textArr[1].trim()
+        }
```

Comment:
```suggestion
        const [ids, importStmt] = component.split(':')
        if (!ids.trim()) throw new Error('No component id')
        return {
          ids: ids.split(',').map(id => id.trim()),
          importStmt: importStmt.trim()
        }
```

类似改改吧，找不到 `identifiers` 的时候丢个错误出来最好